src/Realm.C

```diff
@@ -1928,7 +1942,46 @@ Realm::create_output_mesh()
       if (fileid++ > 0) oname += "-s" + fileid_ss.str();
     }
 
-    resultsFileIndex_ = ioBroker_->create_output_mesh( oname, stk::io::WRITE_RESULTS, *outputInfo_->outputPropertyManager_);
+
+    if(!outputInfo_->catalystFileName_.empty()||
```

Comment:
Yes, the implementation either writes to Catalyst or to standard output but not to both. In Sierra, we duplicate a results output block in the input deck, and have one block for Catalyst and one block for Exodus II output, if desired. In Nalu you can only have one output section in the input deck?